ui/lib/pki/addon/components/parsed-certificate-info-rows.hbs

```diff
@@ -6,4 +6,12 @@
       @formatDate={{field.formatDate}}
     />
   {{/let}}
-{{/each}}
\ No newline at end of file
+{{/each}}
```

Comment:
After discussing with Alex, opted to render banner below the details because it's unlikely we'll come across un-parsable certs and the information is just supplemental